recipe/meta.yaml

```diff
@@ -146,18 +143,18 @@ outputs:
         - idds-client >=0.11.0
         - idds-doma >=0.11.0
         - iminuit >=2
-        - lmfit <1.0.3
+        - lmfit
         - lsstdesc.coord
-        - matplotlib-base !=3.6.0,<3.7.0
+        - matplotlib-base !=3.6.0
```

Comment:
Wait, I thought that display_matplotlib was breaking with 3.7.0 and later?  DM-38089

Reply:
Thanks, that's why I wanted a second pair of eyes.  Some of these pins/unpins got messy in the rebase.